client/src/pages/Home/FooterNav/FooterNav.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { GrMail } from "react-icons/gr";
+import "./FooterNav.css";
+
+export function FooterNav() {
+	return (
+		<div id="footernav">
+			<div className="footernav_container">
+				<div className="nav_container">
+					<div className="nav-box">
+						<p className="title">About</p>
+						<a href="http://example.com/">Out Story</a>
+						<a href="http://example.com/">Team</a>
+					</div>
+					<div className="nav-box">
+						<p className="title">Resources</p>
+						<a href="http://example.com/">Timetables</a>
+						<a href="http://example.com/">Referrals</a>
+						<a href="http://example.com/">Useful Links</a>
+					</div>
```

Comment:
can you fix the href value, if you do not have the link yet please use '#' instead of wrong url